source/renderer/app/stores/index.js

```diff
@@ -23,7 +23,7 @@ export const storeClasses = {
   addresses: AddressesStore,
   blockConsolidation: BlockConsolidationStore,
   app: AppStore,
-  networkStatus: NetworkStatusStore,
+  daedalusDiagnostics: DaedalusDiagnosticsStore,
```

Comment:
@DeeJayElly please revert the name of the store 🙏

Reply:
@nikolaglumac done